packages/adena-extension/src/components/manage-token/manage-token-list-item/manage-token-list-item.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { ManageTokenListItemWrapper } from './manage-token-list-item.styles';
+import { ManageToken } from '@components/manage-token/manage-token-list/manage-token-list';
+import ManageTokenListItemBalance from '@components/manage-token/manage-token-list-item-balance/manage-token-list-item-balance';
+import Toggle from '@components/common/toggle/toggle';
+
+export interface ManageTokenListItemProps {
+  token: ManageToken;
+  onToggleActiveItem: (tokenId: string) => void;
+}
+
+const ManageTokenListItem: React.FC<ManageTokenListItemProps> = ({ token, onToggleActiveItem }) => {
+  const {
+    main,
+    tokenId,
+    logo,
+    name,
+    balanceAmount,
+    activated
+  } = token;
+
+  return (
+    <ManageTokenListItemWrapper onClick={() => onToggleActiveItem(tokenId)}>
```

Comment:
Is the behavior the same when the item is clicked as when the toggle is clicked?

Reply:
Only the toggle button event exists.
The click event has been removed.